### PR TITLE
clarify published catalog URI

### PR DESCRIPTION
**Pull request by kwin**

### CODE REVIEW

**Comment by fbricon** [comment]
@kwin can you please sign-off your commit and either [sign](https://accounts.eclipse.org/user/1227/eca/3.1.0) the Eclipse Foundation ECA or use an email already associated to an ECA?

File: README.md [hunk 1]
```diff
@@ -6,14 +6,17 @@ As a normal Maven user in Eclipse there is nothing special you have to do. If yo
 
 # Testing the M2E Discovery Catalog as a User
 
-If you know that one of the connectors you use has been updated in the staging catalog then you can try the new connector by specifying the staged catalog during the startup of Eclipse. by updating the eclipse.ini file in your Eclipse installation:
+If you know that one of the connectors you use has been updated in the staging catalog then you can try the new connector by specifying the released catalog during the startup of Eclipse by updating the eclipse.ini file in your Eclipse installation:
```
Comment (fbricon) [inline README.md]: "staging catalog" would need to be updated too

Comment (kwin) [reply README.md]: Done

**Comment by kwin** [comment]
> @kwin can you please sign-off your commit and either [sign](https://accounts.eclipse.org/user/1227/eca/3.1.0) the Eclipse Foundation ECA or use an email already associated to an ECA?

Done now. Please check again.

**Comment by fbricon** [comment]
ECA check passes but you still need to sign-off your commit

**Comment by laeubi** [comment]
> ECA check passes but you still need to sign-off your commit

As far as I know sign-off is no longer required..

**Comment by fbricon** [comment]
> > ECA check passes but you still need to sign-off your commit
> 
> As far as I know sign-off is no longer required..

You're right, I forgot

**Comment by fbricon** [comment]
Thanks @kwin

**Pull request merged**